api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -139,7 +139,7 @@
 				<constraints nullable="false" />
 			</column>
 			<column name="verification_status" type="varchar(50)" >
-				<constraints nullable="false"/>
+				<constraints nullable="true"/>
```

Comment:
This should be it separate pull request because it is a followup for another ticket.

Reply:
Oh really? I'll implement that right away.

Reply:
Done @dkayiwa